src/poly/commitment/verifier.rs

```diff
@@ -32,14 +32,14 @@ impl<'a, C: CurveAffine> Guard<'a, C> {
     /// scalars and points.
     pub fn use_challenges(mut self) -> MSM<'a, C> {
         let s = compute_s(&self.challenges_sq, self.allinv * &self.neg_z1);
-        self.msm.add_to_g(&s);
-        self.msm.add_to_h(self.neg_z1);
+        self.msm.add_to_g_scalars(&s);
```

Comment:
`add` -> `append`

Reply:
It isn't appended, it's actually added to the scalar.

Reply:
right, i think we want `add_term` -> `append_term`.